src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.transaction;
+
+import seedu.address.model.person.Person;
+import seedu.address.model.person.UniquePersonList;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class Transaction {
+
+    private final Person payer;
+    private final Double amount;
+    private final String description;
+    private final UniquePersonList payees;
+
+    public Transaction(Person payer, Double amount, String description, UniquePersonList payees) {
+        this.payer = payer;
+        this.amount = amount;
+        this.description = description;
+        this.payees = payees;
+    }
+
+    public Transaction(Person payer, Double amount, String description, Set<Person> payeesToAdd) {
```

Comment:
Why did you create an other constructor using Set ?

Reply:
I supposed is to prevent same payment added to the same payee twice or more

Reply:
I wasn't sure how we wanted to construct the Transaction object. Do we want to create a UniquePersonList each time and pass it in, or make a set instead? If the set constructor is redundant I can remove it.